src/components/HelloWorld.vue

```diff
@@ -8,7 +8,7 @@
     <input v-model="message" placeholder="Address 2" /><br />
     <input v-model="message" placeholder="City" />
     <select v-model="selected">
-      <option disabled value="selet any one">State</option>
+      <option disabled value="">State</option>
```

Comment:
What is the '**disabled**' attribute functionality? Can you please explain.

Reply:
ya, so basically it will disable the starting selection. in this case 'state'. which ofcource is just for reference purpose.

Reply:
Yup.  👍